### PR TITLE
Modale a11y

### PR DESCRIPTION
**Pull request by AlexiaAlarza**
## Description

Add accessibility on modal and sidepanel component.

TO DO
- [x] add attribute `aria-hidden="true"` on body when modal is open

-----

Optionally, technical or more in-depth description for reviewers.
Keep an empty line under your text, as well as the 5 lines that follow it.

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1917/storybook

File: packages/ng/modal/src/lib/modal-panel.component.html [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
+	aria-modal="true"
+	[attr.aria-labelledby]="titleModal"
+>
+	<div role="document">
+		<div class="lu-modal-header" tabindex="-1" cdkFocusInitial>
+			<button type="button" class="lu-modal-header-close" (click)="dismiss()" [luTooltip]="closeLabel">
+				<span aria-hidden="true" class="lucca-icon icon-cross"></span>
+				<span class="u-mask">{{ closeLabel }}</span>
+			</button>
+			<h1 id="titleModal" class="lu-modal-header-title">{{ title$ | async }}</h1>
```
Comment (GuillaumeNury) [inline packages/ng/modal/src/lib/modal-panel.component.html:15]: Sometimes, we have multiple opened modals. We should have a uniq generated `id` ?

File: packages/ng/sidepanel/src/lib/sidepanel-panel.component.html [hunk 2]
```diff
@@ -1,27 +1,36 @@
...
+	aria-modal="true"
+	[attr.aria-labelledby]="titleSidepanel"
+>
+	<div role="document">
+		<div class="lu-modal-header" tabindex="-1" cdkFocusInitial>
+			<button type="button" class="lu-modal-header-close" (click)="dismiss()" [luTooltip]="closeLabel">
+				<span aria-hidden="true" class="lucca-icon icon-cross"></span>
+				<span class="u-mask">{{ closeLabel }}</span>
+			</button>
+			<h1 id="titleSidepanel" class="lu-modal-header-title">{{ title$ | async }}</h1>
```
Comment (GuillaumeNury) [inline packages/ng/sidepanel/src/lib/sidepanel-panel.component.html:15]: Idem

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1917/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1917/storybook

File: packages/ng/modal/src/lib/modal-panel.component.html (hunk 1 above)
Comment (AlexiaAlarza) [reply packages/ng/modal/src/lib/modal-panel.component.html]: yes the id must be unique

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1917/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1917/storybook

**Review by jeremie-lucca — APPROVED** [review]

**Review by RipripKipkip — APPROVED** [review]

**Pull request merged**